.github/workflows/e2e.yml

```diff
@@ -17,7 +17,7 @@ env:
 jobs:
   e2e:
     # we need to start validation on master push or on /ci comment in PR only
-    if: github.event.issue.pull_request == false || (github.event.issue.pull_request && startsWith(github.event.comment.body, '/ci'))
+    if: github.event_name == 'push' || (github.event_name == 'issue_comment' && startsWith(github.event.comment.body, '/ci'))
```

Comment:
Why do you use `issue_comment` here?
```
issue_comment - Activity related to an issue or pull request comment.
```
We don't want to run this action when someone posted comment in issue

Reply:
it's an event name https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#issue_comment

Reply:
check for pull request [added](https://github.com/dell/csi-baremetal/pull/687/commits/67fea6763931b1cab188afbdbdc21dfc51a8dae5)